EIPS/eip-1193.md

```diff
@@ -312,9 +311,21 @@ The Provider is said to be "disconnected" when it cannot service RPC requests to
 
 ### API
 
-The Provider **MUST** expose the API defined in this section. All API entities **MUST** adhere to the types and interfaces defined in this section.
+The Provider API is specified using TypeScript.
+
+The Provider **MUST** implement and expose the API defined in this section. All API entities **MUST** adhere to the types and interfaces defined in this section.
+
+The Provider **MAY** expose methods and properties not specified in this section.
+
+The Provider **MAY** extend TypeScript `interface` declarations to include properties not listed in this specification.
```

Comment:
I don't think this is necessary for a specification.  Since TypeScript types are erased at runtime, whether the provider does this or not is totally transparent to the app so this is just needless noise in the specification.